pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+            if szSelections.split(",")[0] in self.unclassified_names:
+                self.logger.error("Error in uniting objects.")
+                self._odesign.Undo()
+                self.cleanup_objects()
+                return False
+            elif purge:
+                self.purge_history(objs[0])
+            objs_groups.append(objs[0])
+            remaining -= slice
+            if remaining > 0:
+                unite_list = unite_list[slice:]
+        if remaining > 0:
+            objs_groups.extend(unite_list)
+        self.cleanup_objects()
+        if len(objs_groups) > 1:
+            return self.unite(objs_groups, purge=purge)
+        self.logger.info("Union of {} objects has been executed.".format(num_objects))
+        return self.convert_to_selections(unite_list[0], False)
+
+    @pyaedt_function_handler()
+    def clone(self, objid):
+        """Clone objects from a list of object IDs.
+
+        Parameters
+        ----------
+        objid : list
+            List of object IDs.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        List
+            List of names of objects cloned when successful.
+
+        References
+        ----------
+
+        >>> oEditor.Copy
+        >>> oEditor.Paste
+        """
+        self.copy(objid)
+        new_objects = self.paste()
+        return True, new_objects
+
+    @pyaedt_function_handler()
+    def copy(self, object_list):
+        """Copy objects to the clipboard.
+
+            Parameters
+            ----------
+            object_list : list
+                List of objects (IDs or names).
+
+            Returns
+            -------
+            list
+                List of names of the objects copied when successful.
+
+        References
+        ----------
+
+        >>> oEditor.Copy
+        """
+        # convert to string
+
+        try:
+            selections = self.convert_to_selections(object_list)
+            vArg1 = ["NAME:Selections", "Selections:=", selections]
+            self.oeditor.Copy(vArg1)
+            return selections
+        except AttributeError:
+            self.logger.error("Unable to copy selections to clipboard.")
+            return None
+
+    @pyaedt_function_handler()
+    def paste(self):
+        """Paste objects from the clipboard.
+
+        Returns
+        -------
+        list
+            List of passed objects.
+
+        References
+        ----------
+
+        >>> oEditor.Paste
+        """
+        self.oeditor.Paste()
+        new_objects = self.add_new_objects()
+        return new_objects
+
+    @pyaedt_function_handler()
+    def intersect(self, theList, keep_originals=False, **kwargs):
+        """Intersect objects from a list.
+
+        Parameters
+        ----------
+        theList : list
+            List of objects.
+        keep_originals : bool, optional
+            Whether to keep the original object. The default is ``False``.
```

Comment:
```suggestion
            Whether to keep the original objects. The default is ``False``.
```
Added an "s"  to object because the parameter name uses plural (originals).